src/Jss.js

```diff
@@ -27,7 +27,11 @@ declare var __VERSION__: string
 
 const defaultPlugins = rulesPlugins.concat([observablesPlugin, functionsPlugin])
 
+let instanceCounter = 0
+
 export default class Jss {
+  id = instanceCounter++
```

Comment:
Is it equivalent to having it in the constructor?

Reply:
yes, its new property initializers syntax in es-something